### DISCUSSION TITLE
something else then colors

### DISCUSSION THREAD

**Discussion by obovoid** [menip/godot_speech_to_text#10] [Q&A] [answered]
Is it possible to recognize anything else then colors? Maybe basicly starting with hello world (recognized as two words).
So that you can do sentences and work with them. If it is already possible with the kws file, then I would not know how, since I don't understand the "slash" code behind each color anyways.
Glad that I could find this project, it is by far the best arround for godot.

**Comment by menip** [comment, marked as answer]
If you haven't already, take a look through https://samuraisigma.github.io/godot-docs/doc/community/tutorials/misc/speech_to_text.html; in particular, look at the `Necessary files for STT` section.

The slash separates a phrase from a detection threshold, as described here: https://cmusphinx.github.io/wiki/tutoriallm/#keyword-lists.
